services/sentry/lib/eventReducer.js

```diff
@@ -12,6 +12,13 @@ function reduce(channel, initialAggr, ev) {
 		aggr.events.IMPRESSION = mergeImpressionEv(initialAggr.events.IMPRESSION, ev, channel)
 	}
 
+	if (ev.type === 'IMPRESSION_WITH_COMMISSION') {
+		const { earners } = ev
+		earners.forEach(earner => {
```

Comment:
you did this in a very elegant way, but why don't you just do `ev.earners`, you don't need that destructure here